amazon-kinesis-client-multilang/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>amazon-kinesis-client-pom</artifactId>
     <groupId>software.amazon.kinesis</groupId>
-    <version>2.2.11-SNAPSHOT</version>
+    <version>2.2.12</version>
```

Comment:
Let's keep this snapshot version. We only need to update this version during release process